Makefile

```diff
@@ -2086,11 +2078,11 @@ endif
 metallb_config_cleanup: export NAMESPACE=metallb-system
 metallb_config_cleanup: ## deletes the IPAddressPools and l2advertisement resources
 	$(eval $(call vars,$@,metallb))
-	oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/ipaddresspools.yaml
-	oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/l2advertisement.yaml
-	oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/bgppeers.yaml
-	oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/bgpadvertisement.yaml
-	oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/bgpextras.yaml
+	-oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/ipaddresspools.yaml
+	-oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/l2advertisement.yaml
+	-oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/bgppeers.yaml
+	-oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/bgpadvertisement.yaml
+	-oc delete --ignore-not-found=true -f ${DEPLOY_DIR}/bgpextras.yaml
```

Comment:
Is `-` prepended here in case the YAMLs don't exist on the filesystem?

Reply:
Yes, if the files don't exist the command fails.
And since the cleanup is called by `metallb_config`, it fails on first execution unless we ignore the error.